Makefile

```diff
@@ -7,18 +7,18 @@ RUNC_BUILD_PATH=/go/src/github.com/opencontainers/runc/runc
 RUNC_INSTANCE=runc_dev
 COMMIT=$(shell git rev-parse HEAD 2> /dev/null || true)
 RUNC_LINK=$(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
-export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
+export GOPATH:=$(CURDIR)/Godeps/_workspace
 
 .PHONY=dbuild
 
-all:
-ifneq ($(RUNC_LINK), $(wildcard $(RUNC_LINK)))
-	ln -sfn $(CURDIR) $(RUNC_LINK)
-endif
-	go build -ldflags "-X main.gitCommit=${COMMIT}" -tags "$(BUILDTAGS)" -o runc .
+all: $(RUNC_LINK)
+	go build -i -ldflags "-X main.gitCommit=${COMMIT}" -tags "$(BUILDTAGS)" -o runc .
```

Comment:
`-i` seems useful, but I don't see it widely used, is it safe to use when depending on packages that will change?